src/tunacode/core/agents/agent_components/orchestrator/tool_dispatcher.py

```
@@ -152,10 +160,67 @@ def has_tool_calls(parts: list[Any]) -> bool:
+    has_tool_calls: bool
+    used_fallback: bool
+
+
+# ---------------------------------------------------------------------------
+# Collection: structured tool calls
+# ---------------------------------------------------------------------------
+
+
+async def _collect_structured_tool_calls(
+    parts: list[Any],
+    node: Any,
```

Ruff (0.14.14)

[warning] Unused function argument: `node`

(ARG001)